index.js

```diff
@@ -1,5 +1,5 @@
-const DEV = process.env.NODE_ENV && process.env.NODE_ENV !== 'production'
-if (DEV) {
+const IS_DEV_ENV = process.env.NODE_ENV !== 'production'
```

Comment:
In the frontend we do:
```js
const DEV = process.env.NODE_ENV
  ? process.env.NODE_ENV !== 'production'
  : true
```

Generally we want to default to `production`. It's a potential security issue to accidentally run in development mode. It's a inconvenience to accidentally production in development.